packit.yaml

```diff
@@ -9,6 +9,18 @@ actions:
     - tools/webpack-jumpstart --wait --rebase
     - make XZ_OPT=-0 dist
     - sh -c 'echo cockpit-*.tar.xz'
+specfile_path: cockpit.spec
+srpm_build_deps:
```

Comment:
Cool! That may also allow us to drop some hacks which avoid some build dependencies.

Reply:
yup, exactly!

Although I personally dislike that a lot of these deps are a duplicate of BuildRequirements from the spec file. But, in order to extract them from the spec, we would need to process the spec, which means triggering action commands, which means running them in our sandbox... Or create a custom solution in collaboration with Copr.